cmd/soroban-cli/src/commands/network/container/logs.rs

```diff
@@ -23,9 +26,10 @@ pub struct Cmd {
 }
 
 impl Cmd {
-    pub async fn run(&self) -> Result<(), Error> {
+    pub async fn run(&self, global_args: &global::Args) -> Result<(), Error> {
+        let printer = print::Print::new(global_args.quiet);
```

Comment:
one last occurrence. :)

```suggestion
        let print = print::Print::new(global_args.quiet);
```

Reply:
Ah, how did i miss that?! Thanks! :)